### DISCUSSION TITLE
Suppress comma after authors in a private communication?

### DISCUSSION THREAD

**Discussion by jsdodge** [retorquere/zotero-better-bibtex#2415] [Q&A] [answered]
Hello, I would like to cite a private communication, using the following format:

A. Brown and C. Davis (private communication)

Currently I have this in my collection as an "E-mail" item type with the authors "Brown, A." and "Davis, C." and "tex.note=(private communication)" in the "Extra" field. However, this adds a comma  after the author names:

A. Brown and C. Davis, (private communication)

Can anyone recommend a way to format this entry in Zotero that will suppress the comma?

**Comment by retorquere** [comment, marked as answer]
I think you can best ask on https://tex.stackexchange.com/ on how to do this given your bib(la)tex entry, then we can work on making BBT produce whatever changes they propose.

**Comment by jsdodge** [comment]
Thanks—I'll follow the solution given [here](https://tex.stackexchange.com/questions/87272/bibtex-or-biblatex-biber-private-communication-entry).

**Reply by retorquere** [reply]
And you are able to generate this (with sufficient ease) with BBT?
